_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -3323,6 +3323,12 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
         }
     }
 
+    if ((par.mfx.CodecProfile != MFX_PROFILE_AVC_HIGH) && IsOn(extDdi->Transform8x8Mode))
```

Comment:
It's incorrect. If Profile is not set by app explicitly (i.e. CodecProfile == 0), enabling transform 8x8 will always cause error. Please check that CodecProfile is set (not 0) before checking that it's equal to MFX_PROFILE_AVC_HIGH.

Reply:
Done